ForumProject/users/admin.py

```diff
@@ -2,8 +2,20 @@
 
 from .models import CustomUser, UserInvestor, UserStartup, UserRoleCompany
 
-# Register your models here.
-admin.site.register(CustomUser)
-admin.site.register(UserInvestor)
-admin.site.register(UserStartup)
-admin.site.register(UserRoleCompany)
+
+class CustomUserAdmin(admin.ModelAdmin):
+    list_display = ['id', 'email', 'first_name', 'last_name', 'is_active']
+    list_editable = ['is_active']
+admin.site.register(CustomUser, CustomUserAdmin)
+
+class UserInvestorAdmin(admin.ModelAdmin):
+    list_display = ['customuser', 'investor', 'investor_role_id']
+admin.site.register(UserInvestor, UserInvestorAdmin)
```

Comment:
Consider using `select_related` or `prefetch_related` in the `get_queryset` method to optimize database access for foreign key fields
```
class UserInvestorAdmin(admin.ModelAdmin):
    def get_queryset(self, request):
        return super().get_queryset(request).select_related('customuser', 'investor')
```

Reply:
As above